pkg/apps/strategy/recreate/recreate_test.go

```diff
@@ -143,7 +144,8 @@ func TestRecreate_initialDeployment(t *testing.T) {
 
 	config := appstest.OkDeploymentConfig(1)
 	config.Spec.Strategy = recreateParams(30, "", "", "")
-	deployment, _ = appsutil.MakeDeployment(config, legacyscheme.Codecs.LegacyCodec(legacyscheme.Registry.GroupOrDie(kapi.GroupName).GroupVersions[0]))
+	deployment, _ = appsutil.MakeDeploymentV1(config, legacyscheme.Codecs.LegacyCodec(legacyscheme.Registry.GroupOrDie(kapi.GroupName).
```

Comment:
I'd prefer to check errors here (and everywhere else) but this is partially preexisting

Reply:
all that code is pre-existing, prefer to address that as follow up

Reply:
it's fine (technically the function called is different but I get your point)